IBPSA/Fluid/Interfaces/PartialTwoPortVector.mo

```diff
@@ -11,6 +11,16 @@ partial model PartialTwoPortVector "Partial component with two ports, one of whi
           property_T=293.15,
           X_a=0.40)
           "Propylene glycol water, 40% mass fraction")));
+
+  constant Integer port_a_x = -100
+    "x-coordinate of port_a center";
+  constant Integer port_a_y = 0
+    "y-coordinate of port_a center";
+  constant Integer port_b_x = 100
+    "x-coordinate of port_b center";
+  constant Integer port_b_y = 0
+    "y-coordinate of port_b center";
+
```

Comment:
Did you consider a partial model (not only for this)? `PartialPort` with `port_a` and `Medium` and `PartialTwoPort` and `PartialTwoPortVector` extending those?

Reply:
I would prefer not to. Although this would avoid (little) code duplication between `PartialTwoPort` and `PartialTwoPortVector` it would add an inheritance level in interface classes that already derive from each other. For instance, one looking at a simple actuator model such as `Dampers.Exponential` already has to navigate across `PartialDamperExponential`→`PartialResistance`→`PartialTwoPortInterface`→`PartialTwoPort` to travel the whole definition. Adding a new layer that does not abstract different physics may be more cumbersome than helpful.

Reply:
I agree. I got the impression that you prefer inheritance over code repetition at the LBNL, thats why I asked.